src/actions/actions.ts

```diff
@@ -5287,3 +5288,287 @@ class ActionOverrideCmdAltUp extends BaseCommand {
     return vimState;
   }
 }
+
+
+// Only register the EasyMotion actions if the configuration is set
+if (Configuration.getInstance().easymotion) {
+  class BaseEasyMotionCommand extends BaseCommand {
+    public getMatches(position: Position, vimState: VimState): EasyMotion.Match[] {
+      throw new Error("Not implemented!");
+    }
+
+    public getMatchPosition(match: EasyMotion.Match, position: Position, vimState: VimState): Position {
+      return match.position;
+    }
+
+    public processMarkers(matches: EasyMotion.Match[], position: Position, vimState: VimState) {
```

Comment:
You're right about it belonging in `EasyMotion` object, but then I would have to pass in the `getMatchPosition` as a callback in order to map the matches differently on each action (e, T, t have different match mappings)